### DISCUSSION TITLE
Vorta does not appear in Command-tab list on Mac

### DISCUSSION THREAD

**Discussion by rosslaird** [borgbase/vorta#1616] [General]
On Mac (Ventura, 13.2.1), Vorta (.8.10) is not shown in the Command-Tab (or alt-tab) list — at least for me. Therefore, I can only foreground the Vorta window by clicking on the icon in the menu bar, then clicking on "Vorta for Borg Backup." Is this intentional? When I have a backup running, it would be good to be able to foreground the active Vorta window with a single gesture rather than two clicks.

**Comment by rosslaird** [comment]
In reviewing the list of apps that show up — or do not show up — in the command-tab list, it looks like native apps always show up, whereas electron apps show up sometimes (Obsidian does, but Loom does not). This may or may not be related.

**Reply by m3nu** [reply]
Vorta is hidden from the Dock and the Cmd-tab list because it's meant to just run in the background and do it's thing. So it's not a full app you would use from the Dock.

You can access it from the menu bar in the top-right.

**Reply by rosslaird** [reply]
That's fine. Just thought I'd check. Thanks for the info.
